v2.0/src/Components/Pages/HomePage/EventHomepageSection.js

```diff
@@ -47,7 +47,7 @@ class Events extends React.Component {
         );
 
         const location = event.location;
-        const img = event.image.url ? event.image.url : defaultImg;
+        const img = event.image && event.image.url ? event.image.url : defaultImg;
```

Comment:
i would even start with event &&

Reply:
The code would have blown up 5 times already before it got to that line.